### PR TITLE
Added ServiceNow-GetGroups command

### PR DESCRIPTION
**Pull request by dem-evgeny**
New servicenow-GetGroups command

### CODE REVIEW

File: Integrations/integration-ServiceNow.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 commonfields:
-  id: ServiceNow
+  id: ServiceNow2
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: fix in all places

File: Integrations/integration-ServiceNow.yml [hunk 2]
```diff
@@ -529,6 +528,33 @@ script:
...
+
+    var SNGetGroups = function(name){
+        var parameters={};
+
+        var ticket_type = "sys_user_group";
+        var path = "table/"+ticket_type + encodeToURLQuery(parameters);
+        res = sendRequest('GET',path).obj;
+
+        var md = '## ServiceNow groups\n';
+        md += 'Id|Name|Description|Email|Active|Manager|Parent\n';
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: pls use tableToMarkdown

File: Integrations/integration-ServiceNow.yml [hunk 3]
```diff
@@ -529,6 +528,33 @@ script:
         return dicRes;
     }
 
+
+    var SNGetGroups = function(name){
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: make sure you use this argument to bring only group if argument is provided

File: Integrations/integration-ServiceNow.yml [hunk 4]
```diff
@@ -529,6 +528,33 @@ script:
...
+            md += stringEscapeMD(res.result[i].description, true) + '|';
+            md += res.result[i].email + '|';
+            md += res.result[i].active + '|';
+            if (res.result[i].manager) {
+                md += res.result[i].manager.value + '|';
+            }
+            md += res.result[i].parent + '\n';
+        }
+
+        return {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res, "HumanReadable": md, "EntryContext": {}};
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: put in context something like 
ServiceNowGroups [{
   GroupID: id
   GroupName: name
},
...]

File: Integrations/integration-ServiceNow.yml [hunk 5]
```diff
@@ -1385,5 +1493,7 @@ script:
       description: Filename of uploaded file to override the existing file name in
         entry
     description: Upload new file
+  - name: servicenow-get-groups
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: description?
argument? description for the argument?

File: Integrations/integration-ServiceNow.yml (hunk 3 above)
Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: Fixed

File: Integrations/integration-ServiceNow.yml (hunk 2 above)
Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: Changed

File: Integrations/integration-ServiceNow.yml (hunk 4 above)
Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: Added

File: Integrations/integration-ServiceNow.yml (hunk 1 above)
Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: Fixed

File: Integrations/integration-ServiceNow.yml [hunk 6]
```diff
@@ -1385,5 +1508,11 @@ script:
...
+    arguments:
+    - name: name
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: i think we should not have the name mandatory, and if absent - return all groups (including to context) like you did before

**Comment by dorsha** [comment]
@gshriki / @dem-evgeny - what about this PR?

File: Integrations/integration-ServiceNow.yml (hunk 6 above)
Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: Changed

**Comment by dem-evgeny** [comment]
@dorsha I have some conflicts, can't understand how to handle them. Can you please help me on this?

**Comment by dem-evgeny** [comment]
duplicate https://github.com/demisto/content/pull/1032

**Pull request closed without merging**